src/starplot/styles/helpers.py

```diff
@@ -25,8 +25,11 @@ def decorator(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             style = kwargs.get("style")
+            style_args = {x:kwargs[x] for x in kwargs if x[:7] == "style__"}
```

Comment:
can you change the dictionary comprehension to:

`{kw: value for kw, value in kwargs.items() if kw.startswith("style__")}`

I think that makes it easier to read, and startswith is safer (at least in terms of code maintenance, cause then we don't have to worry about the number 7 being wrong)